source/deploy/mobile-faq.rst

```diff
@@ -117,6 +117,14 @@ The following payload shows an example of the json that is transmitted to the pu
         "is_id_loaded": true
     }
 
+where the following definitions are applied:
+
+- ``ack_id``: An ephemeral identifier generated per notification that determines whether the notification sent was received by the device (using same method that generates identifiers to the rest of the models in the server). This information is available in the ``notifications.log`` file on the Mattermost server. The ``ack_id`` is only used for receipt delivery from the mobile app to the Mattermost server to confirm whether the notification sent was received. 
+- ``server_id``: A server identifier created on the server, called ``DiagnosticId``. In the future, this value will be used in the mobile app (for multi-server support) to identify which server the notification belongs to.
+- ``device_id``: The token that APNs and FCM return when you allow the device to receive notifications. So when the user logs into Mattermost, Mattermost sends this ``device_id`` to attach it to the session. If the session is terminated, the ``device_id`` is no longer present in the server database because the session record is removed. When the user logs back in, the ``device_id`` is registered again with the same value because the identifier is specific to the device. This value won't be the same across apps or devices owned by the same person, but will be the same for each session the user creates from the same app on the same device.
+- ``version``: Tells the mobile app how data is structured so it can parse it properly. Current value is ``v2``.
+- ``is_id_loaded``: (Mattermost Enterprise only) When true, the mobile app look for the contents of the notification on the server because those details are not part of the payload. 
```

Comment:
are we going to add the explanation for the other ones or not needed?

Reply:
@enahum - Would you be open to providing explanations for the outstanding ones? They include: platform, post_id, category, message, badge, channel_id, type, sender_id